apps/website/src/components/Navbar.tsx

```
@@ -12,12 +12,19 @@ interface NavbarProps {
 export function Navbar({}: NavbarProps) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)